src/extension.ts

```diff
@@ -752,7 +757,10 @@ async function continueEditing(outline: DocumentOutline, useContext: boolean, se
 				userEndLineEol
 			);
 		}).finally(
-			() => completion.dispose()
+			() => {
+				completion.dispose();
+				completionIndicator.dispose();
```

Comment:
According to [the sample code](https://github.com/microsoft/vscode-extension-samples/blob/main/decorator-sample/src/extension.ts#L11), `vscode.TextEditorDecorationType` instances are constant and reused.
How safe is it to `dispose` each time?

Reply:
From https://github.com/microsoft/vscode-extension-samples/issues/22, https://github.com/microsoft/vscode-discussions/discussions/907#discussioncomment-7462028, and https://code.visualstudio.com/api/references/vscode-api#TextEditorDecorationType, `dispose`ing instances is the only way to do so.
[It might be inefficient](https://github.com/microsoft/vscode-discussions/discussions/907#discussioncomment-7466345) but I'm not sure about another way.

Reply:
Thank you for the code examples.
I now understand that it is safe to use `dispose`